lib/config/configcompilercontext.cpp

```diff
@@ -39,7 +40,13 @@ void ConfigCompilerContext::OpenObjectsFile(const String& filename)
 	m_ObjectsPath = filename;
 
 	std::fstream *fp = new std::fstream();
-	m_ObjectsTempFile = Utility::CreateTempFile(filename + ".XXXXXX", 0600, *fp);
+	try {
+		m_ObjectsTempFile = Utility::CreateTempFile(filename + ".XXXXXX", 0600, *fp);
+	} catch (const std::exception& e) {
+		Log(LogCritical, "cli", DiagnosticInformation(e, false));
+		Log(LogCritical, "cli", "Could not create temporary objects file. Exiting...");
```

Comment:
I think it is better to have only one line with "Could not create ...: <error message>".